3_train/src/train.py

```diff
@@ -286,18 +281,20 @@ def fit(epochs, model, loss_func, opt, train_dl, valid_dl, verbose=False):
     def verbose_print(*args):
         # Only print if verbose is true
         if verbose:
-            print(*args)
+            print(*args, flush=True)
```

Comment:
I needed it to get updates as the model ran. Otherwise, I got no feedback until everything had finished, so there was no way to tell how close it was to finishing!